Tella/Data/Networking/APICall/URLRequest.swift

```diff
@@ -20,11 +20,46 @@ extension WebRepository {
             return URLSession(configuration: configuration)
                 .dataTaskPublisher(for: request)
                 .requestJSON()
-        } catch _ {
+        } catch (let error) {
+            print(error.localizedDescription)
             return Fail<Value, APIError>(error: APIError.invalidURL)
                 .eraseToAnyPublisher()
         }
     }
+    // TODO: Change the name
+    func callReturnsHeaders<Value>(endpoint: any APIRequest) -> AnyPublisher<(Value, HTTPURLResponse?), APIError>
```

Comment:
It will return the headers that are server sends back when it responds to our API call as per old implementation of `call` method it will only return on object either the header object or the body object and parse it according to it I think so can you please check it once and if it will work with the previous `call` method then please send me the code snippet its just 2 3 lines so it will not be a big trouble for you.

Reply:
@rob-horizontal-iOS I've made the change and I pushed to develop, so could you rebase `uwazi` feature onto `develop` and get the updated function `getAPIResponse` and apply it